src/components/export/export.js

```diff
@@ -156,12 +156,28 @@ export default class Export extends Component {
             ? Object.entries(WALLET_COMPONENT.errors).filter(([k, v]) => v.isBad)[0]?.[1].message
             : null
 
-        const addresses = Object.fromEntries(Object.entries(STORAGE.addresses)
-            .map(([k, v]) => {
-                const account = new ArgsAccount(v)
-                errors[k].validOrNull(account);
-                return [k, account]
-            }));
```

Comment:
Even though this const is not used, it will update the state of the errors, try to make a valid proposal, then enter an invalid address like ".sputnik-dao.near". Now if you enter a valid address again, the propose button will not turn green as it thinks errors.dao is still bad.